index.js

```diff
@@ -347,27 +416,27 @@ const styles = StyleSheet.create({
     flexWrap: 'wrap',
   },
   textInput: {
-    height: 36,
```

Comment:
By removing the fixed height, the text input field overlaps with tags in other lines.
<img width="361" alt="screen shot 2017-09-09 at 9 58 17 am" src="https://user-images.githubusercontent.com/2797531/30240816-76eff0b2-9545-11e7-87ab-790431148375.png">